mujoco_warp/_src/math.py

```diff
@@ -234,19 +228,15 @@ def closest_segment_point(a: wp.vec3, b: wp.vec3, pt: wp.vec3) -> wp.vec3:
 
 
 @wp.func
-def closest_segment_point_and_dist(
-  a: wp.vec3, b: wp.vec3, pt: wp.vec3
-) -> Tuple[wp.vec3, wp.float32]:
+def closest_segment_point_and_dist(a: wp.vec3, b: wp.vec3, pt: wp.vec3) -> Tuple[wp.vec3, wp.float32]:
```

Comment:
can the return type be `Tuple[wp.vec3, float]`?

Reply:
yes, and i'll add a TODO to kernel analyzer to check return types